module/tools/RoboCupConfiguration/src/RoboCupConfiguration.cpp

```diff
@@ -388,18 +388,34 @@ namespace module::tools {
 
     std::string RoboCupConfiguration::user_input() {
         // Read characters until we see either esc or enter
-        std::stringstream chars;
+        std::string input;
+        int cursor_pos = 0;
 
         // Keep reading until our termination case is reached
         while (true) {
             auto ch = getch();
             switch (ch) {
-                case 27: return "";
+                case 27: return "";  // Escape key
                 case '\n':
-                case KEY_ENTER: return chars.str(); break;
+                case KEY_ENTER: return input;
+                case KEY_BACKSPACE:
+                case 127:  // ASCII code for backspace
```

Comment:
Why are we doing key backspace then the ascii for backspace via fallthrough? Should it be just `KEY_BACKSPACE`?